Samples/Samples/View/CalendarPage.xaml

```diff
@@ -8,9 +8,10 @@
                  xmlns:essentials="clr-namespace:Xamarin.Essentials;assembly=Xamarin.Essentials"
                  x:Class="Samples.View.CalendarPage"
                  x:DataType="viewmodels:CalendarViewModel"
-                 Title="Calendars">
+                 Title="Calendar">
   <views:BasePage.Resources>
     <ResourceDictionary>
+     <x:String x:Key="DateDisplayFormatter">{0:dd/MM/yy hh:mm tt}</x:String>
```

Comment:
Not fussed about this too much, as this is a sample but this format resource is the same as the one on the calendar event page, therefore we should re-use it instead of creating two of them

Reply:
Moved to App.xaml as that seems to be the only place they store 'globalised' resources